apps/microtvm/arduino/template_project/microtvm_api_server.py

```diff
@@ -78,7 +80,13 @@ class BoardAutodetectFailed(Exception):
     ),
     server.ProjectOption(
         "arduino_cli_cmd",
-        required=["build", "flash", "open_transport"],
+        required=["generate_project", "build", "flash", "open_transport"]
+        if not ARDUINO_CLI_CMD
```

Comment:
can you bring this ternary (foo if bar else baz) into a separate line or parenthesize it so it indents?

Reply:
done